### DISCUSSION TITLE
Supabase not supporing Like and Distinct operator in my project

### DISCUSSION THREAD

**Discussion by awsomedesigner** [supabase/supabase#19943] [General]
Hello, 

We are trying supabase for our development but we face some issue with the supabase. First of all your platform didn't support distinct functionality releated to db tablels columns. also when we are trying to fetch some data with the like operator it's not work properly. In like operator when we give the substring then it did'nt give the related data, but when we give the exact value in like with %% then it's give the data that matches value of rows.

**Comment by GaryAustin1** [comment]
Not clear if you are referring to the REST API client here or SQL directly.  Certainly those work in SQL directly.
like and ilike in the REST API seem to work fine for most, maybe show exactly what issue you are having.  Normally for partials you put the partial word inside % %.  So %art% would find the word partial.
Distinct is not supported in the REST API but there are some new features that might help: https://supabase.com/blog/postgrest-12  See aggregate functions there.
Normally when you reach the limits of the REST API's you use Postgres Views or rpc calls to Postgres functions.
